boards/arm/arduino_zero/doc/arduino_zero.rst

```diff
@@ -0,0 +1,126 @@
+.. _arduino_zero:
+
+Arduino/Genduino Zero
+################
```

Comment:
Underline for headings needs to be at least as long as the title above it (add more ####)

Reply:
Done.